static/js/reactions.js

```diff
@@ -142,15 +142,17 @@ exports.reaction_navigate = function (e, event_name) {
     var first_emoji = get_emoji_at_index(0);
     var selected_emoji = get_selected_emoji();
     var selected_index = find_index_for_emoji(selected_emoji);
-
     // special cases
     if (event_name === 'down_arrow') {
         if ($('.emoji-popover-filter').is(':focus') && first_emoji) { // move down into emoji map
             $(first_emoji).focus();
         }
     } else if (event_name === 'up_arrow') {
         if (selected_emoji && selected_index < 6) { // move up into reaction filter
-            $('.emoji-popover-filter').focus();
+            setTimeout(function () {
+                $('.emoji-popover-filter').focus();
+                $('.emoji-popover-filter').caret(Infinity);
```

Comment:
Please add a comment here saying that we always go the end, even if the browser is smart like Firefox.